with-snowflake/README.md

```diff
@@ -0,0 +1,315 @@
+# StepZen Example: `with-snowflake`
+
+## Introduction
+
+This project builds a GraphQL API connected to your Snowflake warehouse using StepZen.
+We will use the `@dbquery` directive to generate our GraphQL schema.
```

Comment:
This repo does not allow me to suggest, hmm.  We don't use @dbquery to generate schema...

We will use the StepZen custom directive `@dbquery` to indicate that the data is resolved from a database.

Or

We will use the StepZen custom directive `@dbquery` to generate the SQL and connections required to resolve the requested data.